tests/git/test_git_commands.py

```diff
@@ -53,3 +54,30 @@ def test_fetch(mock_connector, runner, mock_ctx):
 
     assert result.exit_code == 0, result.output
     assert ctx.get_query() == "alter git repository repo_name fetch"
+
+
+@mock.patch("snowflake.connector.connect")
+def test_copy(mock_connector, runner, mock_ctx):
```

Comment:
Should we have integration tests?

Reply:
I'm on it in another PR

Reply:
https://github.com/snowflakedb/snowflake-cli/pull/865